ja-jp/Content/index.htm

```diff
@@ -219,7 +219,7 @@
                                             <div class="dropdown" data-test="menuBarCategory" id="documentationMenu" data-mc-conditions="MultiProdPublish.DocMenu">
                                                 <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>
                                                 <div class="dropdown-content" data-test="menuBarItems">
-                                                    <div><a href="../../ja-jp/Content/index.htm" data-test="menuBarLink">製品ドキュメント</a>
+                                                    <div><a href="../../en-us/Content/index.htm" data-test="menuBarLink">製品ドキュメント</a>
```

Comment:
This is unfortunate. Need to add a search/replace for the top-level index link to our en2ja.json file.